openid-connect-4-credential-issuance-1_0.md

```diff
@@ -719,8 +736,23 @@ HTTP/1.1 400 Bad Request
 
 ## Deferred Credential Endpoint
 
-[TBD]
-An endpoint where the RP uses the acceptance token to fetch the credential.
+This endpoint is used to issue a credential previously requested at the credential endpoint in case the Issuer was not able to immediately issue this credential. 
+
+### Deferred Credential Request
+
+This is a HTTP POST request, which accepts an acceptance token as only parameter. The acceptance token MUST be sent as access token in the HTTP header as shown in the following example.
+
+```
+POST /credential_deferred HTTP/1.1
+Host: server.example.com
+Content-Type: application/x-www-form-urlencoded
+Authorization: BEARER 8xLOxBtZp8
+
+```
```

Comment:
no need to define CIBA style push and ping..?

Reply:
no, why?